### ISSUE TITLE
adjust train/val/test times to accomodate metabolism data

### ISSUE THREAD

**Issue by jsadler2** [USGS-R/drb-do-ml#59]
In the baseline model, our partitions have been as follows:

partition | start | end
-|-|-
train| 1980-01-01| 2017-01-01
validation| 2017-01-01| 2019-01-01
test| 2019-01-01 | 2022-01-01

These will have to change because the metabolism data only goes from Fall 2007 to Fall 2016 (#57)

**Comment by jsadler2** [comment]
@galengorski - if we shortened the validation and test time periods to one year, would that give us enough data to do the information theory metrics? I remember you saying that they can be data hungry and you said 200+ points would be needed. So one year is 200+, but just wanted to double check.

**Comment by galengorski** [comment]
I am thinking we might end up assessing functional performance across the full data set, although it would be interesting to the train/val/test sets separately to see how they differ. The estimates that I am going off of are based on this figure [paper here](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2008WR007279):

![MI_Data](https://user-images.githubusercontent.com/32860978/158430994-e834947a-139f-474e-ba8c-3f5651c387e4.PNG)

the y-axis on the left panel is mutual information between two variables, you can can see that the value (solid line) doesn't really stabilize until about 500, and the same is true for transfer entropy in the right panel. I think we'll have to end up doing some testing to make sure we are making stable estimates. With the current data coverage, I don't think we'll be able to use these metrics separately on validation sets at individual sites

**Comment by jsadler2** [comment]
Closing with #90
